charts/newrelic-logging/static/lowdatamodedefaults.yaml

```diff
@@ -0,0 +1,52 @@
+fluent-bit.conf: |
```

Comment:
I would recommend not to define a separate file just for the `lowDataMode`. The only major change is in the `FILTER`s section, and an environment flag that is read in the `OUTPUT` section.